Update Harvest Containers

Sabre142 opened the pull request.

Sabre142 committed:
Update Harvest Containers

* Add message
 - Patch update available at Harvest Containers xOBSE:
   https://www.nexusmods.com/oblivion/mods/52317/

MacSplody commented on masterlist.yaml:
```diff
@@ -2219,6 +2219,10 @@ plugins:
     msg: [ *alreadyInOOO ]
+  - name: 'Harvest\[Containers\]\.es[mp]'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Harvest Containers xOBSE](https://www.nexusmods.com/oblivion/mods/52317/)' ]
```
Use instead would be a better here, and you could add a condition to check for OBSE.

Sabre142 replied:
Harvest Containers already requires OBSEv15 so I'll check for xOBSE 22.6 or greater.

Sabre142 committed:
Update Harvest Containers

* Add message
 - Use Harvest Containers xOBSE instead.
 - Condition requires xOBSE 22.6 or greater.

sibir-ine reviewed (approved):
Thanks!
